package.json

```diff
@@ -15,36 +22,71 @@
     "@types/node": "^16.11.39",
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
+    "@types/react-redux": "^7.1.25",
+    "axios": "^1.3.5",
+    "bootstrap": "^5.2.3",
     "classnames": "^2.3.1",
-    "eslint-plugin-jest": "^27.0.1",
+    "eslint-plugin-jest": "^27.2.1",
     "formik": "^2.2.9",
```

Comment:
Not sure both MUI and Prime are needed.  Prime is new to me...does it have "market share" among clients that use React?  MUI has had market share for some time, although admittedly these things are changing all the time.